README.md

```diff
@@ -1,3 +1,135 @@
 # scats
-
 Selenium Cucumber automated testing system
+
```

Comment:
could we update README.md on how to do that? A simple hello world project setup will be very helpful.